app/views/_mobile-service-sdk-docs.html

```diff
@@ -0,0 +1,25 @@
+<div class="col-md-6 service-instance-getting-started" ng-repeat="sdkInfo in ctrl.serviceSdkInfo">
+  <div class="col-md-12">
+    <img class="logo" ng-src="{{sdkInfo.logo}}">
```

Comment:
Do you need an `ng-if`, in case there isn't a logo?  Or will there always be a logo path provided?